Makefile

```diff
@@ -0,0 +1,75 @@
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+CI ?= false
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+CRUISE_CONTROL_VERSION ?= 2.5.113
+CRUISE_CONTROL_UI_GIT_REF ?= b1208a6f020c21ff967297814c2e893eed3f3183
```

Comment:
Question: Should this always be the latest commit in the [cruise-control-ui repo](https://github.com/linkedin/cruise-control-ui)? Or does it need to match the corresponding Cruise Control version has (if so, where can we find this info)?

Reply:
I guess for testing purposes locally this can get moved forward, but I would vote the committed value should be the latest version, tag for CC, commit for CC UI.

But Chris CMIIW.

Reply:
Also the UI is developed separately from CC and there releases are not related.

Sadly, the UI is rarely released, therefore the UI is pinned to an exact commit on the `master` branch of its repo. Pinning the version makes it reproducible(-ish). Also we cannot assume that the latest commit always works, so manual testing is needed.

Reply:
Understood, thanks for the info @pregnor @chrisgacsal